server/src/utils/csvReader.ts

```diff
@@ -59,9 +63,7 @@ function readRowValues(
         try {
           for (const [header, headerId] of headersMap.entries()) {
             const word = row[header];
-            if (word) {
-              await createWord(gameId, headerId, wordId, word);
-            }
+            await createWord(gameId, headerId, wordId, word);
```

Comment:
we should insert blank string or null or undefined if thats how the user wants to represent this cell